WooCommerce/src/androidTest/assets/mocks/mappings/wpcom/woocommerceandroid_product_details_linked_products_banner.json

```
@@ -0,0 +1,21 @@
+{
+    "request": {
+        "method": "GET",
+        "urlPathPattern": "/wpcom/v2/experiments/0.1.0/assignments/woocommerceandroid/(.*)",
+        "queryParameters": {
+            "experiment_names": {
+                "equalTo": "woocommerceandroid_explat_aa_test_202208,woocommerceandroid_product_details_linked_products_banner"
```

This is an assumption at this point but potentially, there could be different experiment names here, what do you think about using a wildcard so that when there are new experiments we don't have to update this?

```suggestion
                "matches": "woocommerceandroid_explat_aa_test(.*)"
```

---

Your assumption makes perfect sense, the experiments won't last forever. I addressed this in a594a73be2716ccc11966e66c3aef8241454de55 (waiting for the 🟢 dot).

FYI, I went for using a sort of logical AND with two `contains` instead of using `matches` with a wildcard, since there are two experiment mentioned in the request. I could not have the `Logical AND` from [WireMock docs](https://wiremock.org/docs/request-matching/) (ho header links, but you can search for `Logical AND and OR`) to work for this case, for some reason. What I used seems to behave exactly like AND.